.gitignore

```diff
@@ -15,3 +15,10 @@ __pycache__
 cdk.out
 .idea/
 cdk.context.json
+
+
+# venv
+
+bin/
+lib/python3.11/
+pyvenv.cfg
```

Comment:
Why are you ignoring these, do you have any secrets here @bwighane ?

Reply:
Those files are specific to my environment, the where generated using `pip  -m venv .`
For instance the contents of pyvenv.cfg are 

home = /Library/Frameworks/Python.framework/Versions/3.11/bin
include-system-site-packages = false
version = 3.11.1
executable = /Library/Frameworks/Python.framework/Versions/3.11/bin/python3.11
command = /Library/Frameworks/Python.framework/Versions/3.11/bin/python3 -m venv /Users/ibwighane/LastMileHealth/dl-etl-1

Which would not make sense to the codebase on your machine.

if you are running python 3.11.x you can recreate them by running the command so that you have your own versions.